fog/api/src/conversions.rs

```diff
@@ -66,3 +74,39 @@ impl TryFrom<&ingest_common::IngestSummary> for mc_fog_types::ingest_common::Ing
         Ok(result)
     }
 }
+
+impl TryFrom<crate::view::MultiViewStoreQueryResponse>
+    for mc_fog_types::view::MultiViewStoreQueryResponse
+{
+    type Error = ConversionError;
+    fn try_from(
+        mut proto_response: crate::view::MultiViewStoreQueryResponse,
+    ) -> Result<Self, Self::Error> {
+        let store_responder_id =
+            FogViewStoreUri::from_str(proto_response.get_store_uri())?.responder_id()?;
+        let result = mc_fog_types::view::MultiViewStoreQueryResponse {
+            encrypted_query_response: proto_response.take_query_response().into(),
+            store_responder_id,
+            store_uri: proto_response.get_store_uri().to_string(),
+            status: proto_response.get_status().into(),
+            block_range: BlockRange::from(proto_response.take_block_range()),
+        };
+        Ok(result)
+    }
+}
+
+impl From<crate::view::MultiViewStoreQueryResponseStatus> for MultiViewStoreQueryResponseStatus {
+    fn from(proto_status: crate::view::MultiViewStoreQueryResponseStatus) -> Self {
+        match proto_status {
+            crate::view::MultiViewStoreQueryResponseStatus::SUCCESS => {
```

Comment:
Good call. I added an `UNKNOWN` status to be 0, which means it will default to this. I'd rather make an explicit type for this scenario than reuse another enum value.